pkg/runtime.v2/core/trainingruntime.go

```diff
@@ -84,6 +84,21 @@ func (r *TrainingRuntime) NewObjects(ctx context.Context, trainJob *kubeflowv2.T
 func (r *TrainingRuntime) buildObjects(
 	ctx context.Context, trainJob *kubeflowv2.TrainJob, jobSetTemplateSpec kubeflowv2.JobSetTemplateSpec, mlPolicy *kubeflowv2.MLPolicy, podGroupPolicy *kubeflowv2.PodGroupPolicy,
 ) ([]client.Object, error) {
+
+	info := r.getRuntimeInfo(ctx, trainJob, jobSetTemplateSpec, mlPolicy, podGroupPolicy)
+	if err := r.framework.RunEnforceMLPolicyPlugins(info); err != nil {
+		return nil, err
+	}
+	err := r.framework.RunEnforcePodGroupPolicyPlugins(trainJob, info)
+	if err != nil {
+		return nil, err
+	}
+	return r.framework.RunComponentBuilderPlugins(ctx, info, trainJob)
+}
+
+func (r *TrainingRuntime) getRuntimeInfo(
```

Comment:
```suggestion
func (r *TrainingRuntime) runtimeInfo(
```

or 

```suggestion
func (r *TrainingRuntime) ComputeRuntimeInfo(
```

Could you follow the effective Go style guides?

- https://go.dev/doc/effective_go#Getters
- https://google.github.io/styleguide/go/decisions#getters